core/ErrorQueue.h

```diff
@@ -1,10 +1,12 @@
 #ifndef SORBET_ERROR_QUEUE_H
 #define SORBET_ERROR_QUEUE_H
 
+#include "GlobalState.h"
 #include "common/concurrency/ConcurrentQueue.h"
 #include "core/ErrorFlusher.h"
 #include "core/ErrorFlusherStdout.h"
 #include "core/ErrorQueueMessage.h"
+#include "core/GlobalState.h"
```

Comment:
This file mentioned `GlobalState` in signatures before this change. Why does it need to include it now?

I'm particularly curious because there's a bit of a chicken and egg problem: `GlobalState` keeps track of an `ErrorQueue` and of course ErrorQueue has functions which accept `GlobalState` arguments.

I'd love if we could delete this include, and if we still need it for something I'd love to figure out how to address that with a more targeted change.